AirCasting/CoreData/ExternalSessionEntity+CoreDataProperties.swift

```diff
@@ -45,6 +45,13 @@ extension ExternalSessionEntity {
         
         return abStreams.isEmpty ? allStreams : abStreams
     }
+    
+    func defaultStreamSelection() -> MeasurementStreamEntity? {
```

Comment:
I would put it in an extension which lives near the View file that this function is used in. This is a model file, it should not contain view-related stuff.

Applies to 2 other instances too (`SessionEntity` and `ExternalSessionWithBlaBla`)

Reply:
Sure, sounds right